spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.feature 'User Show', type: :feature do
+  let(:user) { create(:user) }
+  let!(:first_post) { create(:post, author: user) }
+
+  before do
+    visit user_path(user)
+  end
+
+  scenario "viewing a user's details" do
+    expect(page).to have_content(user.name)
+    expect(page).to have_selector("img[src$='path_to_default_profile_pic.jpg']")
+    expect(page).to have_content(user.bio) if user.bio
+    expect(page).to have_content("#{user.posts.count} posts")
+    user.posts.limit(3).each do |post|
+      expect(page).to have_content(post.title)
+    end
+  end
```

Comment:
- [ ] Please you are missing a test that tests that you can see a button that lets you view all of a user's posts. :pray: